docker-compose.yml

```diff
@@ -213,6 +213,7 @@ services:
       - _APP_DATABASE_SHARED_NAMESPACE
       - _APP_FUNCTIONS_CREATION_ABUSE_LIMIT
       - _APP_CUSTOM_DOMAIN_DENY_LIST
+      - _APP_GEO_SECRET=$_APP_GEO_SECRET
```

Comment:
What is this secret - why does both appwrite-geo and appwrite container need it?

I was thinking it's some external API key, but then I would expect only appwrite-geo needs it.